### PR TITLE
Add join table visits

### PR DESCRIPTION
**Pull request by topeogunleye**
Added Changes
- Created a table named vets
- Created a "join table" called specializations for species and vets
- Created a "join table" called visits for animals and vets
- Inserted data for vets
- Insert data for specialties
- Insert the following data for visits

### Write queries to answer the following

- Who was the last animal seen by William Tatcher?
- How many different animals did Stephanie Mendez see?
- List all vets and their specialties, including vets with no specialties.
- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
- What animal has the most visits to vets?
- Who was Maisy Smith's first visit?
- Details for most recent visit: animal information, vet information, and date of visit.
- How many visits were with a vet that did not specialize in that animal's species?
- What specialty should Maisy Smith consider getting? Look for the species she gets the most.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -79,4 +79,71 @@ UPDATE animals
...
+
+-- Insert the following data for specializations:
+-- Vet William Tatcher is specialized in Pokemon.
+-- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
+-- Vet Jack Harkness is specialized in Digimon.
+
+INSERT INTO specializations (vet_id, species_id) VALUES
+(1, 1),
+(3, 2),
+(4, 2);
```
Comment (mmsesay) [inline data.sql]: - [x] `Vet Stephanie Mendez` should have two animals. Pokemon and Digimon. By the look of things you only insert for Digimon. Please insert for Pokemon.

Example:
` (3, 1)`

File: data.sql [hunk 2]
```diff
@@ -79,4 +79,71 @@ UPDATE animals
...
+((SELECT id FROM animals WHERE name = 'Squirtle'), (SELECT id FROM vets WHERE name = 'Vet Stephanie Mendez'), '2019-09-29'),
+((SELECT id FROM animals WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Vet Jack Harkness'), '2020-10-03'),
+((SELECT id FROM animals WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 
+'Vet Jack Harkness'), '2020-11-04'),
+((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Vet Maisy Smith'), '2019-01-24'),
+((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Vet Maisy Smith'), '2020-05-15'),
+((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Vet Maisy Smith'), '2020-02-27'),
+((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Vet Maisy Smith'), '2020-08-03'),
+((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Vet Stephanie Mendez'), '2020-05-24'),
+((SELECT id FROM animals WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'Vet William Tatcher'), '2021-01-11');
```
Comment (mmsesay) [inline data.sql]: - [x] The animal name for this last record should be `Blossom` you used `Agumon`. Let's try to update it please.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @topeogunleye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @topeogunleye, 

## APPROVED 🟢 

Great implementation of all the requirements in this milestone and the changes that were suggested by the previous reviewer. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**